Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+
+    agent { 
+        docker {
+	    image 'fluidity/spudbase:bionic'
+            label 'dockerhost'
+        } 
+    }
+    environment {
+        OMPI_MCA_btl = '^openib'
+    }
+    stages {
+        stage('Configuring') {   
+            steps { 
+                sh './configure --prefix=/home/spud' 
+            }
+        }    
+        stage('Building for python2') {       
+            steps { 
+                sh 'make -j'
+                sh 'make doc'
```

Comment:
Leave for the "building documentation" stage?

Reply:
Good spot - thank you!